public/content/developers/docs/oracles/index.md

```diff
@@ -402,6 +402,9 @@ There are multiple oracle applications you can integrate into your Ethereum dapp
 
 **[API3 DAO](https://www.api3.org/)** - _API3 DAO is delivering first-party oracle solutions that deliver greater source transparency, security and scalability in a decentralized solution for smart contracts_
 
+**[Supra](https://supraoracles.com/)** - Supra is a novel, high-throughput Oracle & IntraLayer: A vertically integrated toolkit of cross-chain solutions (data oracles, asset bridges, automation network, and more) that interlink all blockchains, public (L1s and L2s) or private (enterprises). Supra provides decentralized oracle price feeds that can be used for on-chain and off-chain use-cases such as spot and perpetual DEXes, lending protocols, and payments protocols. Supra’s oracle chain and consensus algorithm makes it the fastest-to-finality oracle provider, with layer-1 security guarantees. The pull oracle has a sub-second response time. Aside from speed and security, Supra’s rotating node architecture gathers data from 40+ data sources and applies a robust calculation methodology to get the most accurate value. The node provenance on the data dashboard also provides a fully transparent historical audit trail. Supra’s Distributed Oracle Agreement (DORA) paper was accepted into ICDCS 2023, oldest distributed systems conference.
```

Comment:
Any chance we could trim this down to be one sentence, similar to the others in the list?
Also, looks like that URL forwards now, so we should use https://supra.com/

Reply:
Hi @wackerow, thanks for your reply. I just submitted the changes you requested. Kindly let me know when it is merged. Thank you!